lib/node_modules/@stdlib/blas/base/zscal/lib/ndarray.js

```diff
@@ -48,35 +45,17 @@ var imag = require( '@stdlib/complex/float64/imag' );
 * zscal( 3, za, zx, 1, 0 );
 * // zx => <Complex128Array>[ -2.0, 6.0, -2.0, 14.0, -2.0, 22.0 ]
 */
-function zscal( N, za, zx, strideZX, offsetZX ) {
-	var view;
-	var re1;
-	var im1;
-	var re2;
-	var im2;
-	var sx;
+function zscal( N, za, zx, strideX, offsetX ) {
 	var ix;
 	var i;
 
 	if ( N <= 0 ) {
 		return zx;
 	}
-	// Reinterpret the input array as a real-valued array of interleaved real and imaginary components:
-	view = reinterpret( zx, 0 );
```

Comment:
@aman-095 Please revert this refactoring. I just recently updated this to avoid complex number instantiation. Don't simply copy-paste from `cscal`.